File: 867-transpose-matrix/867-transpose-matrix.py
import numpy as np

class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        m, n = len(matrix), len(matrix[0])
        res = [[0] * m for _ in range(n)]
        for j in range(n):
            for i in range(m):
                res[j][i] = matrix[i][j]
        return res

File: 867-transpose-matrix/867-transpose-matrix.cpp
#include <vector>

class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
        vector<vector<int>> vvi;
        for (int i = 0; i < matrix[0].size(); ++i) {
            vector<int> vi;
            for (int j = 0; j < matrix.size(); ++j)
                vi.push_back(matrix[j][i]);
            vvi.push_back(vi);
        }
        return vvi;
    }
};